bitbots_hcm/bitbots_hcm/hcm_dsd/actions/wait_for.py

```diff
@@ -13,6 +10,7 @@ class WaitForIMUStartup(AbstractActionElement):
 
     def __init__(self, blackboard, dsd, parameters=None):
         super().__init__(blackboard, dsd, parameters)
+        self.blackboard: HcmBlackboard
```

Comment:
Don't you want to create `AbstractHCMActionElement` like for the Decisions

Reply:
Good point, but not that important at the moment imo..